.ci/lib.sh

```diff
@@ -464,6 +464,23 @@ gen_clean_arch() {
 	# make sure cargo and rustup home dir are removed
 	info "remove cargo and rustup home dir"
 	sudo rm -rf ~/.cargo ~/.rustup
+
+	# make sure coco artifacts are removed
+	info "remove coco artifacts"
+	sudo rm -rf /opt/confidential-containers
+
+	# make sure to clean up binaries
+	unlink /opt/confidential-containers/share/ovmf
+	sudo rm -f /usr/share/ovmf
+
+	unlink /opt/confidential-containers/share/td-shim
+	sudo rm -f /usr/share/td-shim
+
+	unlink /opt/confidential-containers/share/tdvf
+	sudo rm -f /usr/share/tdvf
+
+	unlink /opt/confidential-containers/bin/qemu-system-x86_64
+	sudo rm -f /usr/bin/qemu-system-x86_64
```

Comment:
You're missing:
*  `./opt/confidential-containers/bin/qemu-system-x86_64-tdx` which seems to be linked as `/usr/bin/qemu-system-x86_64-tdx`
* `/opt/kata/libexec/virtiofsd`, which is linked as `/usr/libexec/virtiofsd`
`